docs/cli/packages.md

```diff
@@ -144,7 +144,26 @@ Options:
 ### Install a library
 
 You can then install any package from this list by executing `qx package
-install <URI>`. The URI takes the form `github_user/repository[/path]`: the first
+install <URI>`. It has the following options:
+
+```
+  --release, -r    Use a specific release tag instead of the tag of the latest 
+                   compatible release
+  [string]
```

Comment:
Is this `[string]` in the wrong place? Should it be underneath or just after `compatible release`?